mappings/net/minecraft/util/math/Direction.mapping

```diff
@@ -64,9 +72,11 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_23224 getRotationQuaternion ()Lnet/minecraft/class_1158;
 	METHOD method_23225 transform (Lnet/minecraft/class_1159;Lnet/minecraft/class_2350;)Lnet/minecraft/class_2350;
 	METHOD method_23955 getUnitVector ()Lnet/minecraft/class_1160;
+	METHOD values ()[Lnet/minecraft/class_2350;
 	CLASS class_2351 Axis
 		FIELD field_11050 BY_NAME Ljava/util/Map;
 		FIELD field_11053 name Ljava/lang/String;
+		FIELD field_23780 ALL [Lnet/minecraft/class_2350$class_2351;
```

Comment:
```suggestion
		FIELD field_23780 VALUES [Lnet/minecraft/class_2350$class_2351;
```

Reply:
I saw the `Direction.ALL` and tried to be consistent.